README.md

```diff
@@ -54,31 +54,46 @@ To output a recursive listing of the object direcotry, with the default being a
 
 ## ptcp
 
-Ptcp is a cp-like tool that can copy files in and out of the Pairtree structure.Unlike Linux's cp, the default is recursive. 
+Ptcp is a cp-like tool that can copy files and folders in and out of the Pairtree structure. Unlike Linux's cp, the default is recursive. Ptcp's defualt behavior will also not overwrite files or directories if they already exist at the specificed location. Instead, it will add `.x` (x being an integer that starts from 1) to the path. 
 
-When an ENV PAIRTREE_ROOT, the basic command is
+This cp tool behaves similarly to the unix cp in relation to directories. This means that when you are copying from `SRC` to `DEST` if the folder does not exist, the folder will be created and only the contents of the src will be copied into `DEST`. If the folder does exist, the folder and its contents from `SRC` will be moved into `DEST`. When wanting to copy into a subpath or new path in the pairtree, the `-n` flag will need to be used and is detailed below. At the moment the ability to copy from one pairtree to another pairtree is not available. 
+
+When an ENV PAIRTREE_ROOT is set, the basic command is
     
-    ptcp [ID] [/path/to/output/]
+    ptcp [SRC] [DEST]
+
+so to copy from a paitree ID to an output path 
+
+    ptcp [ID] [/path/to/output]
 
 When an ENV PAIRTREE_ROOT is not set, the command is 
 
-    ptcp [PT_ROOT] [ID] [/path/to/output/]
+    ptcp -p [PT_ROOT] [/path/to/output] [ID]
```

Comment:
Why are the ID and /path/to/output flipped here? It reads like it's just the same as the above example, but demonstrating how to explicitly set the pairtree_root path. It seems like the example should be the same (i.e., copying an ID/object in the pairtree to the file system, with the extra addition of setting the pt root.

Reply:
You could have another showing going the opposite direction from file system into pairtree ID (without the `-n` below option below) if you want to be comprehensive in the examples.

Reply:
Yes the goal was just to demonstrate setting the pairtree_root path. I'll switch it to be the same.